src/lib/jwt.js

```diff
@@ -1,5 +1,6 @@
+import crypto from 'crypto'
 import jose from 'jose'
-import hkdf from 'futoin-hkdf'
+import thirdPartyHkdf from 'futoin-hkdf'
```

Comment:
maybe we could import this conditionally with require, when we don't use hkdf from crypto?

Reply:
thank you for your reviews!